sickbeard/helpers.py

```diff
@@ -1574,38 +1574,12 @@ def verify_freespace(src, dest, oldfile=None):
 
     logger.log(u"Trying to determine free space on destination drive", logger.DEBUG)
 
-    if platform.system() == 'Windows':
-        import sys
-
-        def disk_usage(path):
-            _, total, free = ctypes.c_ulonglong(), ctypes.c_ulonglong(), ctypes.c_ulonglong()
-            if sys.version_info >= (3,) or isinstance(path, unicode):
-                method = ctypes.windll.kernel32.GetDiskFreeSpaceExW
-            else:
-                method = ctypes.windll.kernel32.GetDiskFreeSpaceExA
-            ret = method(path, ctypes.byref(_), ctypes.byref(total), ctypes.byref(free))
-            if ret == 0:
-                logger.log(u"Unable to determine free space, something went wrong", logger.WARNING)
-                raise ctypes.WinError()
-            return free.value
-
-    elif hasattr(os, 'statvfs'):  # POSIX
-        def disk_usage(path):
-            import subprocess
-            call = subprocess.Popen(["df", path], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            output = call.communicate()[0]
-            return output.split("\n")[1].split()[3]
-
-    else:
-        logger.log(u"Unable to determine free space on your OS")
-        return True
-
     if not ek(os.path.isfile, src):
         logger.log("A path to a file is required for the source. {0} is not a file.".format(src), logger.WARNING)
         return True
```

Comment:
Probably someone that hadn't worked on the project before added the code or something like that.

There's parts of the js where you can see that too as there's code where we already have functions for.